service/src/main/java/com/ge/predix/acs/service/policy/validation/PolicySetValidatorImpl.java

```diff
@@ -83,10 +87,70 @@ public void removeCachedConditions(final PolicySet policySet) {
     @Override
     public void validatePolicySet(final PolicySet policySet) {
         validateSchema(policySet);
+        List<String> obligationIds = validateObligationsAndGetIds(policySet.getObligations());
```

Comment:
Please add unit tests for all new validation functionality

Reply:
unit test are aready created, the will be moved to other class as specified